server/src/test/java/org/spine3/server/stand/StandShould.java

```diff
@@ -511,15 +511,15 @@ public void onCompleted() {
 
     @Test
     public void retrieve_collection_fields_if_required() {
-        requestSampleCustomer(new int[] {Customer.FAMILYINFO_FIELD_NUMBER - 1}, new StreamObserver<QueryResponse>() {
+        requestSampleCustomer(new int[] {Customer.NICKNAMES_FIELD_NUMBER - 1}, new StreamObserver<QueryResponse>() {
```

Comment:
Please check, whether the `StreamObserver` methods have been invoked at all.

Right now it is possible that the observer is not invoked for some reason, the test _should_ fail, but it will not.